Building from source fails on current master w/ GStreamer 1.26 on Linux

It fails like this: 
```
~/Qt/6.8.3/gcc_64/bin/qt-cmake -B build -G Ninja -DCMAKE_BUILD_TYPE=Debug

........

-- Checking for module 'gstreamer-rtsp-1.0'
--   Found gstreamer-rtsp-1.0, version 1.26.0
-- Checking for module 'gstreamer-gl-egl-1.0'
--   Found gstreamer-gl-egl-1.0, version 1.26.0
-- Checking for module 'gstreamer-gl-wayland-1.0'
--   Found gstreamer-gl-wayland-1.0, version 1.26.0
-- Checking for module 'gstreamer-gl-x11-1.0'
--   Found gstreamer-gl-x11-1.0, version 1.26.0
-- Found GStreamer:  (found suitable version "1.26.0", minimum required is "1.20") found components: Core Base Video Gl GlPrototypes Rtsp GlEgl GlWayland GlX11
-- CPM: Adding package gstreamer_good_plugins@1.26.2 (1.26.2 to /home/andre/prog/qgroundcontrol/build/cpm_modules/gstreamer_good_plugins/3bf24ab9f5df6f52a7e3f74774ba46ccfeee555f)
/home/andre/prog/qgroundcontrol/build/cpm_modules/gstreamer_good_plugins/3bf24ab9f5df6f52a7e3f74774ba46ccfeee555f/ext/qt6/vertex.vert -> vertex.vert.qsb exposed as :/org/freedesktop/gstreamer/qml6/vertex.vert.qsb
/home/andre/prog/qgroundcontrol/build/cpm_modules/gstreamer_good_plugins/3bf24ab9f5df6f52a7e3f74774ba46ccfeee555f/ext/qt6/YUV_TRIPLANAR.frag -> YUV_TRIPLANAR.frag.qsb exposed as :/org/freedesktop/gstreamer/qml6/YUV_TRIPLANAR.frag.qsb
/home/andre/prog/qgroundcontrol/build/cpm_modules/gstreamer_good_plugins/3bf24ab9f5df6f52a7e3f74774ba46ccfeee555f/ext/qt6/RGBA.frag -> RGBA.frag.qsb exposed as :/org/freedesktop/gstreamer/qml6/RGBA.frag.qsb
/home/andre/prog/qgroundcontrol/build/cpm_modules/gstreamer_good_plugins/3bf24ab9f5df6f52a7e3f74774ba46ccfeee555f/ext/qt6/YUV_BIPLANAR.frag -> YUV_BIPLANAR.frag.qsb exposed as :/org/freedesktop/gstreamer/qml6/YUV_BIPLANAR.frag.qsb
CMake Error at /home/andre/Qt/6.8.3/gcc_64/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:1843 (message):
  The source file
  '/home/andre/prog/qgroundcontrol/build/src/VideoManager/VideoReceiver/GStreamer/gstqml6gl/.qsb/RGBA.frag.qsb.external'
  was specified with an absolute path and is used in a Qt resource.  Please
  set the QT_RESOURCE_ALIAS property on that source file to a relative path
  to make the file properly accessible via the resource system.
Call Stack (most recent call first):
  /home/andre/Qt/6.8.3/gcc_64/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:2371 (__qt_get_relative_resource_path_for_file)
  /home/andre/Qt/6.8.3/gcc_64/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:406 (_qt_internal_process_resource)
  /home/andre/Qt/6.8.3/gcc_64/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:458 (qt6_add_resources)
  CMakeLists.txt:549 (qt_add_resources)

```

--- a/src/VideoManager/VideoReceiver/GStreamer/gstqml6gl/CMakeLists.txt
+++ b/src/VideoManager/VideoReceiver/GStreamer/gstqml6gl/CMakeLists.txt
@@ -23,7 +23,7 @@ endif()
 ################################################################################
 
 if(GStreamer_VERSION VERSION_GREATER_EQUAL 1.22)
-    # Use Latest Revisions for each minor version: 1.20.7, 1.22.12, 1.24.12, 1.26.2
+    # Use Latest Revisions for each minor version: 1.20.7, 1.22.12, 1.24.12, 1.26.3
     string(REPLACE "." ";" GST_VERSION_LIST ${GStreamer_VERSION})
     list(GET GST_VERSION_LIST 0 GST_VERSION_MAJOR)
     list(GET GST_VERSION_LIST 1 GST_VERSION_MINOR)
@@ -36,7 +36,7 @@ if(GStreamer_VERSION VERSION_GREATER_EQUAL 1.22)
     elseif(GST_VERSION_MINOR EQUAL 24)
         set(GST_VERSION_PATCH 12)
     elseif(GST_VERSION_MINOR EQUAL 26)
-        set(GST_VERSION_PATCH 2)
+        set(GST_VERSION_PATCH 3)
     endif()
 
     set(GST_PLUGINS_VERSION ${GST_VERSION_MAJOR}.${GST_VERSION_MINOR}.${GST_VERSION_PATCH})
@@ -153,7 +153,7 @@ if(EXISTS "${QGC_GST_QT6_PLUGIN_PATH}/resources.qrc")
             VERBATIM
         )
 
-        set_source_files_properties(${RGBA_EXTERNAL} PROPERTIES QT_RESOURCE_ALIAS org/freedesktop/gstreamer/qml6/RGBA.frag.qsb.external)
+        set_source_files_properties(${RGBA_EXTERNAL} TARGET_DIRECTORY ${CMAKE_PROJECT_NAME} PROPERTIES QT_RESOURCE_ALIAS org/freedesktop/gstreamer/qml6/RGBA.frag.qsb.external)
         set(QGC_GSTREAMER_SHADER_RESOURCES "${RGBA_EXTERNAL}" CACHE INTERNAL "")
         # qt_add_resources(gstqml6gl "gstqml6gl_shaders1"
         #     PREFIX "/org/freedesktop/gstreamer/qml6"

